sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -68,9 +71,44 @@ case class ScriptTransformation(
       val errorStream = proc.getErrorStream
       val reader = new BufferedReader(new InputStreamReader(inputStream))
 
-      val (outputSerde, outputSoi) = ioschema.initOutputSerDe(output)
+      // TODO make the 2048 configurable?
```

Comment:
This is a carryover from the old code; I ended up moving a bunch of code around without updating comments.